packages/account/src/BaseSmartAccount.ts

```diff
@@ -47,7 +47,6 @@ export abstract class BaseSmartAccount implements IBaseSmartAccount {
     this.overheads = _smartAccountConfig.overheads;
     this.entryPointAddress = _smartAccountConfig.entryPointAddress ?? DEFAULT_ENTRYPOINT_ADDRESS;
     this.accountAddress = _smartAccountConfig.accountAddress;
-    this.paymaster = _smartAccountConfig.paymaster;
```

Comment:
this should be assigned if provided right

Reply:
Yes, I missed that, removed it for some testing then forgot to add it back.

Reply:
you can resolve it once you push